connectors/src/connectors/zendesk/index.ts

```diff
@@ -133,8 +151,87 @@ export class ZendeskConnectorManager extends BaseConnectorManager<null> {
     internalIds: string[];
     viewType: ContentNodesViewType;
   }): Promise<Result<ContentNode[], Error>> {
-    logger.info({ internalIds }, "Retrieving batch content nodes");
-    throw new Error("Method not implemented.");
+    const brandIds: number[] = [];
+    const brandHelpCenterIds: number[] = [];
+    const brandTicketsIds: number[] = [];
+    const categoryIds: number[] = [];
+    internalIds.forEach((internalId) => {
+      let brandId = getBrandIdFromInternalId(this.connectorId, internalId);
+      if (brandId) {
+        brandIds.push(brandId);
+        return;
+      }
+      brandId = getBrandIdFromTicketsId(this.connectorId, internalId);
+      if (brandId) {
+        brandTicketsIds.push(brandId);
+        return;
+      }
+      brandId = getBrandIdFromHelpCenterId(this.connectorId, internalId);
+      if (brandId) {
+        brandHelpCenterIds.push(brandId);
+        return;
+      }
+      const categoryId = getCategoryIdFromInternalId(
+        this.connectorId,
+        internalId
+      );
+      if (categoryId) {
+        categoryIds.push(categoryId);
+      }
```

Comment:
same as below, would be more clear to extract the type and do a switch case . Also need to ensure all cases are handled correctly (here nothing happens if you're not brand/ticket/helpcenter/category)